cove/lib/common.py

```diff
@@ -93,8 +93,15 @@ def _release_pkg_schema_obj(self):
         return json.loads(self.release_pkg_schema_str)
 
     def deref_schema(self, schema_str):
-        return jsonref.loads(schema_str, loader=CustomJsonrefLoader(schema_url=self.schema_host),
-                             object_pairs_hook=OrderedDict)
+        loader = CustomJsonrefLoader(schema_url=self.schema_host)
+        try:
+            deref_obj = jsonref.loads(schema_str, loader=loader, object_pairs_hook=OrderedDict)
+            # Force evaluation of jsonref.loads here
+            repr(deref_obj)
```

Comment:
I found quite difficult to force evaluation of the `deref_object` here, e.g `dict(jsonref.loads(schema_str ...) )` doesn't seem to work. I saw in the source code for JsonRef that it evaluates on representation, so this works, although I wonder how correct this approach is.